.github/workflows/agent-review-on-demand.yml

```
@@ -311,6 +311,19 @@ jobs:
           PR: ${{ needs.authorize.outputs.pr }}
         run: node .trusted/scripts/agent/rebuttal.mjs read "$PR" --out /tmp/rebuttals.json
 
+      # And the fix agent's own reports, for the same reason as the rebuttals
+      # directly above: this advisory review must reach the SAME verdict the gating
+      # panel would, or a maintainer reading it is being told something the merge
+      # gate does not agree with. Trusted copy, fail-quiet, adjudicated not verified
+      # (see scripts/agent/fix-report.mjs).
+      - name: Read fix-agent reports (align with the gating panel)
+        if: steps.pr.outputs.number != ''
```

actionlint (1.7.12)

[error] property "pr" is not defined in object type {diff: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)